packages/server/migrations/1594758731888-migration.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1594758731888 implements MigrationInterface {
+    name = 'migration1594758731888'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query('CREATE TABLE "email_verification" ("id" SERIAL NOT NULL, "userId" character varying NOT NULL, "emailVerificationToken" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_b985a8362d9dac51e3d6120d40e" PRIMARY KEY ("id"))');
```

Comment:
I'd add CREATE TABLE "email_reset" here and remove previous migration file with it